.github/workflows/buildTest.yml

```diff
@@ -8,81 +8,105 @@ on:
     - readme/**
     - '**/*.md'
 
-  push:
-    paths-ignore:
-    - Bootloader/**
-    - Copy to SD Card root directory to update/**
-    - images/**
-    - readme/**
-    - '**/*.md'
-
```

Comment:
This action no longer needs to run on push/direct commits since it is handled under the Build Binaries action.